src/main.js

```diff
@@ -281,12 +281,13 @@ define('peaks', [
       get: function () {
         var self = this;
 
-        function addSegment (startTime, endTime, editable, color, labelText) {
+        function addSegment (startTime, endTime, editable, color, labelText, id) {
```

Comment:
I can hopefully look at that sometime next week.   How about merging #142 to fix Travis?